archaius-aws/src/main/java/com/netflix/config/sources/AbstractDynamoDbConfigurationSource.java

```diff
@@ -41,12 +45,16 @@
     static final String keyAttributePropertyName = "com.netflix.config.dynamo.keyAttributeName";
     static final String valueAttributePropertyName = "com.netflix.config.dynamo.valueAttributeName";
     static final String endpointPropertyName = "com.netflix.config.dynamo.endpoint";
+    static final String pollingMaxBackOff = "com.netflix.config.dynamo.maxPollingBackOff";
```

Comment:
It would be great if you identified the time unit (i.e. ms) in the variable name, or at the very least put in a comment.  I am assuming it is milliseconds.